scope & closures/ch2.md

```diff
@@ -72,27 +72,27 @@ Não importa o *local* onde uma função é invocada, ou até mesmo *como* é in
 
 O processo de consulta ao escopo léxico ocorre *apenas" em identificadores de primeira classe, como `a`, `b` e `c`. Se você tivesse uma referência para `foo.bar.baz` em um trecho de código, ocorreria uma consulta ao escopo léxico para localizar o identificador `foo`, mas no momento que esta variável é localizada, regras de acesso à propriedades de objetos assumem o comando para resolução das propriedades `bar` e `baz`, respectivamente.
 
-## Cheating Lexical
+## Trapaceando o Léxico
 
-If lexical scope is defined only by where a function is declared, which is entirely an author-time decision, how could there possibly be a way to "modify" (aka, cheat) lexical scope at run-time?
+Se o escopo léxico é de fato definido apenas pelo local onde uma função é declarada, cuja decisão vem do autor do código durante a programação, como pode haver uma maneira de "modificar" (ou trapacear) o escopo léxico em tempo de execução?
```

Comment:
Diferente de `runtime` ou `compile time`, que são termos com valor semântico dentro das Ciências da Computação, `author-time` me parece se tratar de uma terminologia cunhada pelo autor aproveitando-se de um recurso que a língua inglesa proporciona, que é a facilidade de se criar supostos "neologismos" que passam a ter o valor semântico justamente por sua semelhança com outros jargões da mesma área -- neste caso, `runtime` (cuja tradução comum é de fato `tempo de execução`) . A minha escolha para este termo se deu também pelo fato de, curiosamente, encontrar [esta dúvida](http://stackoverflow.com/questions/35686903/author-time-vs-run-time-in-javascript) enquanto procurava a melhor forma de flexibilizar este termo! :)